support-pie/donate.html

```diff
@@ -81,7 +81,7 @@
 					<p>We appreciate and recognize our sponsors in the form of posters, banners and logos, and t-shirts seen by thousands of members of the UC Berkeley community. In addition, sponsors are warmly welcomed and encouraged to attend the competition day.</p>
 
 
-					<p>If you would like more information about the benefits, please refer to <a href="{{ '/assets/sponsors/SponsorshipLevels15-16.pdf' | prepend: site.baseurl }}">Sponsorship Levels 2015-2016.pdf</a></p>
+					<p>If you would like more information about the benefits, please refer to <a href="{{ site.sponsors | prepend: site.baseurl }}">Sponsorship Levels 2015-2016.pdf</a></p>
```

Comment:
... when I click on this, it just refreshes the page (and inspecting yields an empty href) :/